libs/SalesforceAnalytics/SalesforceAnalytics/Classes/Model/SFSDKInstrumentationEventBuilder.m

```diff
@@ -69,7 +69,7 @@ - (SFSDKInstrumentationEvent *) buildEvent {
         errorMessage = @"Mandatory field 'page' not set!";
     }
     if (errorMessage) {
-        [SFSDKAnalyticsLogger w:[self class] format:@"WARNING: Building event failed! REASON: %@", errorMessage];
+        [SFSDKAnalyticsLogger log:[self class] format:@"WARNING: Building event failed! REASON: %@", errorMessage];
```

Comment:
Shouldn't be `warn` (or whatever the method is called to log at log level warning)?